### PR TITLE
.

### PR DESCRIPTION
**Pull request by PaltineanuRares956**

### CODE REVIEW

File: Paltineanu Rares-Mihai/curs10 - 3.4/main.c [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+        }
+
+    }
+}
+
+long combination(int n, int k) {
+
+    long nr = 1;
+
+    if(k == 0) {
```
Comment (Alex-Cosma) [inline Paltineanu Rares-Mihai/curs10 - 3.4/main.c:44]: hint:
``` 
if (k==0 || k ==n) {
    return 1;
}
```

File: Paltineanu Rares-Mihai/curs10 - 3.4/main.c [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+
+    }
+
+    return nr;
+
+}
+
+void takeTheValues(long *real, long *imaginary, char *vect) {
+
+    ///I know that i could have in the struct int real, imaginary; and then read the coefficients, but i wanted to make the things more complicated
```
Comment (Alex-Cosma) [inline Paltineanu Rares-Mihai/curs10 - 3.4/main.c:68]: Don't get used to over-complicating your life, it's not going to get any easier!

**Review by Alex-Cosma — APPROVED** [review]

File: Paltineanu Rares-Mihai/curs10 - 3.4/main.c (hunk 1 above)
Comment (PaltineanuRares956) [reply Paltineanu Rares-Mihai/curs10 - 3.4/main.c:44]: Thanks

File: Paltineanu Rares-Mihai/curs10 - 3.4/main.c (hunk 2 above)
Comment (PaltineanuRares956) [reply Paltineanu Rares-Mihai/curs10 - 3.4/main.c:68]: Okk

**Pull request merged**